pkg/testutils/testconfig/testconfig.go

```diff
@@ -52,6 +53,17 @@ func IPFromTestNet(hostIPending string) string {
 	return *flagTestNet + hostIPending
 }
 
+// IPFromSecondTestNet returns IP from the host network:
+//
+//	IPFromTestNet("11") -> 192.168.100.11
+//	IPFromTestNet("11") -> 2001:0DB9:100::11.
```

Comment:
What is this formatting?

Reply:
Good catch, this is copy paste comment from IPFromTestNet method comment.
The examples just show the output for given parameter.

Updated.

Reply:
To be honest I meant strange tabulation and dot at the end, but it's a nit so I'm not going to stall this PR because of that.